Readme.md

```diff
@@ -528,6 +526,56 @@ cuba.addon.saml.logAllSamlMessages = true
 
 Also, you can observe the details of the implementation in the corresponding [demo project](https://git.haulmont.com/app-components/saml-addon-demo).
 
+## 4.2. Setup signing method for SAML messages <a name="setup-signing-method"></a>
+
+By default opensaml spring component uses sha1 digest algorithm for signing SAML messages. The most convenient way to use different signing message
+is create class in `WEB` module with addition changes in SecurityContext.
```

Comment:
is **to** create **a** class in **the** `web` module with **additional** changes in `SecurityContext`.

Reply:
fixed